install_config/install/docker_registry.adoc

```diff
@@ -398,7 +430,11 @@ $ docker pull docker.io/busybox
 ----
 ====
 
-. Tag the new image with the form `<registry_ip>:<port>/<project>/<image>`:
+. Tag the new image with the form `<registry_ip>:<port>/<project>/<image>`.
```

Comment:
You can also append a tag after `<image>`. If you omit it, it defaults to latest.